packages/create-turbo/src/cli.ts

```diff
@@ -44,8 +48,8 @@ createTurboCli
   --example-path foo/bar
 `
   )
-  .version(cliPkg.version, "-v, --version", "output the current version")
-  .helpOption()
+  .version(cliPkg.version, "-v, --version", "Output the current version")
+  .helpOption("-h, --help", "Display help for command")
```

Comment:
This just fixes a minor casing inconsistency in the help message that has bothered me for a while